### ISSUE TITLE
Audio file shorter than expected from CUE file

### ISSUE THREAD

**Issue by bartdelange** [flacon/flacon#159]
Hi there,

When I tried to split an album it showed me the error mentioned in the title
After closer inspection of the cue file and the timings reported in flacon I noticed that the song lengths were wrong.

Can you please take a look?

<img width="620" alt="image" src="https://user-images.githubusercontent.com/32978882/128406663-d295ba0f-5a16-48c5-86c7-56d0357c8dc9.png">

```cue
PERFORMER "Muse"
TITLE "Showbiz"
FILE "Muse_Showbiz1.flac" WAVE
  TRACK 01 AUDIO
    TITLE "Sunburn"
    PERFORMER "Muse"
    INDEX 01 00:00:00
  TRACK 02 AUDIO
    TITLE "Muscle Museum"
    PERFORMER "Muse"
    INDEX 01 03:55:01
  TRACK 03 AUDIO
    TITLE "Fillip"
    PERFORMER "Muse"
    INDEX 01 08:17:33
  TRACK 04 AUDIO
    TITLE "Falling Down"
    PERFORMER "Muse"
    INDEX 01 12:20:13
  TRACK 05 AUDIO
    TITLE "Cave"
    PERFORMER "Muse"
    INDEX 01 16:55:53
  TRACK 06 AUDIO
    TITLE "Showbiz"
    PERFORMER "Muse"
    INDEX 01 21:42:68
FILE "Muse_Showbiz2.flac" WAVE
  TRACK 07 AUDIO
    TITLE "Unintended"
    PERFORMER "Muse"
    INDEX 01 00:00:00
  TRACK 08 AUDIO
    TITLE "Uno"
    PERFORMER "Muse"
    INDEX 01 03:57:79
  TRACK 09 AUDIO
    TITLE "Sober"
    PERFORMER "Muse"
    INDEX 01 07:36:77
  TRACK 10 AUDIO
    TITLE "Escape"
    PERFORMER "Muse"
    INDEX 01 11:40:11
  TRACK 11 AUDIO
    TITLE "Overdue"
    PERFORMER "Muse"
    INDEX 01 15:12:24
  TRACK 12 AUDIO
    TITLE "Hate This & I'll Love You"
    PERFORMER "Muse"
    INDEX 01 17:40:96
```

**Comment by SokoloffA** [comment]
CUE content looks good. 

Pay attention to the red error icon. If you hover the mouse over it, you will see a list of errors why the program cannot convert the file, something like.
![Error message](https://user-images.githubusercontent.com/854935/128418122-5ed1a1ce-92ff-4191-8f34-9bea9dfef423.png)  

Also you can run the profgam from the console in the debug mode.

If you are using Linux: Open the terminal program (konsole, GNOME Terminal, etc) and type:
```
flacon --debug
```

If you are using MacOS: Run the Terminal application and type:
```
/Applications/Flacon.app/Contents/MacOS/Flacon --debug
```

**Comment by bartdelange** [comment]
The only error it shows is the error in the title. 
![image](https://user-images.githubusercontent.com/32978882/128841413-e444a23c-6ec7-4f45-b17f-68c71064d9cb.png)

It is a 192kHz/24bit flac file, maybe thats a problem?
Debug mode did not show anything special:
![image](https://user-images.githubusercontent.com/32978882/128841904-597af64e-5c57-4d4c-b955-e44a57e83f5f.png)

**Comment by SokoloffA** [comment]
The program supports 192kHz/24bit flac files.
Please check the duration of the source files, and post output of 
```
/Applications/Flacon.app/Contents/MacOS/sox Muse_Showbiz1.flac -n stat
/Applications/Flacon.app/Contents/MacOS/sox Muse_Showbiz2.flac -n stat
```

It would be great if you sent me your files for tests. You can write to me privately at Sokoloff.A@gmail.com

**Comment by SokoloffA** [comment]
I can't diagnose the problem without more information. 
Maybe your problem is related to issue #165 (fixed in the 8.2.0 release). Please test the new version of the program, if the error remains, feel free to open the issue again.

**Comment by petebueh** [comment]
Hi,

I have the same problem with many cue sheets that reference two or more CDs or two-sided Vinyls.
I get the following error message:
"The conversion is not possible.
Audio file shorter than expected from CUE sheet."


Here is an example:

```
REM GENRE E
PERFORMER "Elton John"
TITLE "Goodbye Yellow Brick Road (Deluxe Edition)"
REM REPLAYGAIN_ALBUM_GAIN -7.60 dB
REM REPLAYGAIN_ALBUM_PEAK 1.000000
FILE "Elton John - Goodbye Yellow Brick Road - Disc 1.wav" WAVE
  TRACK 01 AUDIO
    TITLE "Funeral For A Friend /Love Lies Bleeding"
    PERFORMER "Elton John"
    REM REPLAYGAIN_TRACK_GAIN -8.43 dB
    REM REPLAYGAIN_TRACK_PEAK 0.999084
    INDEX 01 00:00:00
  TRACK 02 AUDIO
    TITLE "Candle In The Wind"
    PERFORMER "Elton John"
    REM REPLAYGAIN_TRACK_GAIN -6.36 dB
    REM REPLAYGAIN_TRACK_PEAK 0.984070
    INDEX 01 11:08:47
  TRACK 03 AUDIO
    TITLE "Bennie And The Jets "
    PERFORMER "Elton John"
    REM REPLAYGAIN_TRACK_GAIN -7.32 dB
    REM REPLAYGAIN_TRACK_PEAK 0.998505
    INDEX 01 14:58:48
  TRACK 04 AUDIO
    TITLE "Goodbye Yellow Brick Road"
    PERFORMER "Elton John"
    REM REPLAYGAIN_TRACK_GAIN -7.05 dB
    REM REPLAYGAIN_TRACK_PEAK 0.997162
    INDEX 01 20:23:11
  TRACK 05 AUDIO
    TITLE "This Song Has No Title"
    PERFORMER "Elton John"
    REM REPLAYGAIN_TRACK_GAIN -5.72 dB
    REM REPLAYGAIN_TRACK_PEAK 0.976166
    INDEX 01 23:36:65
  TRACK 06 AUDIO
    TITLE "Grey Seal "
    PERFORMER "Elton John"
    REM REPLAYGAIN_TRACK_GAIN -7.77 dB
    REM REPLAYGAIN_TRACK_PEAK 0.997131
    INDEX 01 26:01:16
  TRACK 07 AUDIO
    TITLE "Jamaica Jerk-Off"
    PERFORMER "Elton John"
    REM REPLAYGAIN_TRACK_GAIN -7.92 dB
    REM REPLAYGAIN_TRACK_PEAK 0.998749
    INDEX 01 30:02:62
  TRACK 08 AUDIO
    TITLE "I've Seen That Movie Too"
    PERFORMER "Elton John"
    REM REPLAYGAIN_TRACK_GAIN -5.27 dB
    REM REPLAYGAIN_TRACK_PEAK 0.997284
    INDEX 01 33:42:51
FILE "Elton John - Goodbye Yellow Brick Road - Disc 2.wav" WAVE
  TRACK 09 AUDIO
    TITLE "Sweet Painted Lady"
    PERFORMER "Elton John"
    REM REPLAYGAIN_TRACK_GAIN -3.81 dB
    REM REPLAYGAIN_TRACK_PEAK 0.996063
    INDEX 01 00:00:00
  TRACK 10 AUDIO
    TITLE "The Ballad Of Danny Bailey (1909-1934)"
    PERFORMER "Elton John"
    REM REPLAYGAIN_TRACK_GAIN -6.70 dB
    REM REPLAYGAIN_TRACK_PEAK 0.997284
    INDEX 01 03:55:37
  TRACK 11 AUDIO
    TITLE "Dirty Little Girl"
    PERFORMER "Elton John"
    REM REPLAYGAIN_TRACK_GAIN -8.10 dB
    REM REPLAYGAIN_TRACK_PEAK 0.999054
    INDEX 01 08:20:29
  TRACK 12 AUDIO
    TITLE "All The Girls Love Alice"
    PERFORMER "Elton John"
    REM REPLAYGAIN_TRACK_GAIN -6.62 dB
    REM REPLAYGAIN_TRACK_PEAK 0.998169
    INDEX 01 13:22:32
  TRACK 13 AUDIO
    TITLE "Your Sister Can't Twist (But She Can Rock 'n' Roll)"
    PERFORMER "Elton John"
    REM REPLAYGAIN_TRACK_GAIN -8.98 dB
    REM REPLAYGAIN_TRACK_PEAK 0.999969
    INDEX 01 18:32:71
  TRACK 14 AUDIO
    TITLE "Saturday Night's Alright For Fighting"
    PERFORMER "Elton John"
    REM REPLAYGAIN_TRACK_GAIN -9.12 dB
    REM REPLAYGAIN_TRACK_PEAK 0.999237
    INDEX 01 21:14:68
  TRACK 15 AUDIO
    TITLE "Roy Rogers"
    PERFORMER "Elton John"
    REM REPLAYGAIN_TRACK_GAIN -7.07 dB
    REM REPLAYGAIN_TRACK_PEAK 0.997467
    INDEX 01 26:11:60
  TRACK 16 AUDIO
    TITLE "Social Disease"
    PERFORMER "Elton John"
    REM REPLAYGAIN_TRACK_GAIN -8.69 dB
    REM REPLAYGAIN_TRACK_PEAK 0.997131
    INDEX 01 30:20:60
  TRACK 17 AUDIO
    TITLE "Harmony"
    PERFORMER "Elton John"
    REM REPLAYGAIN_TRACK_GAIN -6.87 dB
    REM REPLAYGAIN_TRACK_PEAK 0.998535
    INDEX 01 34:04:53
  TRACK 18 AUDIO
    TITLE "Whenever You're Ready (We'll Go Steady)"
    PERFORMER "Elton John"
    REM REPLAYGAIN_TRACK_GAIN -9.46 dB
    REM REPLAYGAIN_TRACK_PEAK 0.999573
    INDEX 01 36:54:12
  TRACK 19 AUDIO
    TITLE "Jack Rabbit"
    PERFORMER "Elton John"
    REM REPLAYGAIN_TRACK_GAIN -5.98 dB
    REM REPLAYGAIN_TRACK_PEAK 0.996735
    INDEX 01 39:46:35
  TRACK 20 AUDIO
    TITLE "Screw You (Young Man's Blues)"
    PERFORMER "Elton John"
    REM REPLAYGAIN_TRACK_GAIN -9.67 dB
    REM REPLAYGAIN_TRACK_PEAK 1.000000
    INDEX 01 41:37:33
  TRACK 21 AUDIO
    TITLE "Candle In The Wind (Acoustic)"
    PERFORMER "Elton John"
    REM REPLAYGAIN_TRACK_GAIN -4.99 dB
    REM REPLAYGAIN_TRACK_PEAK 0.853668
    INDEX 01 46:20:01
```

You asked for the output of

/Applications/Flacon.app/Contents/MacOS/sox /Volumes/DISC_2TB/CD/Elton\ John\ -\ Goodbye\ Yellow\ Brick\ Road/Elton\ John\ -\ Goodbye\ Yellow\ Brick\ Road\ -\ Disc\ 1.wav -n stat

```
Samples read:         210012432
Length (seconds):   2381.093333
Scaled by:         2147483647.0
Maximum amplitude:     0.996460
Minimum amplitude:    -0.999084
Midline amplitude:    -0.001312
Mean    norm:          0.135599
Mean    amplitude:    -0.000000
RMS     amplitude:     0.190660
Maximum delta:         1.831207
Minimum delta:         0.000000
Mean    delta:         0.115529
RMS     delta:         0.165797
Rough   frequency:         6103
Volume adjustment:        1.001
```

/Applications/Flacon.app/Contents/MacOS/sox /Volumes/DISC_2TB/CD/Elton\ John\ -\ Goodbye\ Yellow\ Brick\ Road/Elton\ John\ -\ Goodbye\ Yellow\ Brick\ Road\ -\ Disc\ 2.wav -n stat

```
Samples read:         265641936
Length (seconds):   3011.813333
Scaled by:         2147483647.0
Maximum amplitude:     0.999969
Minimum amplitude:    -1.000000
Midline amplitude:    -0.000015
Mean    norm:          0.143849
Mean    amplitude:    -0.000000
RMS     amplitude:     0.200993
Maximum delta:         1.732849
Minimum delta:         0.000000
Mean    delta:         0.116695
RMS     delta:         0.164939
Rough   frequency:         5759
Volume adjustment:        1.000
```

If I make a cue sheet for each individual WAV-file, everything works correct.

I'm using Flacon 8.2.0

**Comment by SokoloffA** [comment]
I confirm that the program incorrectly parses the time from your CUE file.

I'll fix it soon.

**Comment by SokoloffA** [comment]
The fix is available in release 8.3.0
